csm/plugins/cortx/rgw.py

```diff
@@ -104,7 +100,7 @@ def _build_response(self, operation, response) -> Any:
             except Exception as e:
                 Log.error(f"Error occured while coverting raw api response to\
                     required response: {e}")
-                raise CsmInternalError(const.S3_CLIENT_ERROR_MSG)
+                raise e
```

Comment:
Why are we raising e?

Reply:
this is the private method and its error handling is taken care at calling method.